hazelcast-jet-core/src/test/java/com/hazelcast/jet/TestInClusterSupport.java

```diff
@@ -51,7 +51,7 @@
     protected static final String JOURNALED_CACHE_PREFIX = "journaledCache.";
     protected static final int MEMBER_COUNT = 2;
 
-    protected static JetTestInstanceFactory factory = new JetTestInstanceFactory();
+    protected static JetTestInstanceFactory FACTORY = new JetTestInstanceFactory();
```

Comment:
caps is only used for static final variables (i.e. constants?)

Reply:
Ok, will revert. Truth to be told I've not come across that many non-final static fields in the past and I've never stopped to consciously make this distinction.